modules/ROOT/pages/connect-clients-to-target.adoc

```diff
@@ -201,6 +202,6 @@ For more information, see the following:
 
 Your migration is now complete, and your target cluster is the source of truth for your client applications and data.
 
-When you are ready, you can decommission your origin cluster and {product-proxy}, as these are no longer needed and clean xref:ROOT:rollback.adoc[rollback] is no longer possible.
+When you are ready, you can decommission your origin cluster and {product-proxy} because these are no longer needed and xref:ROOT:rollback.adoc[seamless rollback] is no longer possible.
```

Comment:
This is out of scope, but on line 189, there is an extra `and`
If you migrated to {astra-db} or {hcd-short}, you have the option of using the Data API instead of, or in addition to, a {cass-short} driver.